### PR TITLE
Addresses #587 (balance of testing slots)

### PR DESCRIPTION
**Pull request by kyordhel**
- Updated time diagram for 3-days competition
- Explicitly stated all teams receive same number of testing slots.

### CODE REVIEW

**Comment by benatuts** [comment]
I'm unclear how to interpret point 1 ("Thematic Scenario Blocks") in 3.6.2 with respect to the Morning of Day 2.

On Day 2 morning, are teams free to choose from either thematic scenario, or is it one slot of  Housekeeper followed by one slot of Party Host? If it is the latter, then which scenario is used if there is time remaining in the block to have an additional testing slot?

Thanks!

**Comment by johaq** [comment]
Yes teams can choose from either scenario

File: general_rules/Organization.tex [hunk 1]
```diff
@@ -29,26 +26,26 @@ \subsection{Stage system}\label{rule:stages}
...
-	Testing slots are randomly assigned to teams in each block.
+	\item \textbf{Slots:} In principle, all teams get the same amount of \iterm{testing slots} with a minimum of 2 per block.
+	If there is sufficient unoccupied remaining time for all the teams in a league, a referee can open an extra testing slot, allowing an extra run for all.
 
-	\item \textbf{Tests:} Teams must inform the OC in advance which task(s) will try to solve.
+	\item \textbf{Tests:} Teams must inform the OC in advance which tasks will be tested in each block.
 	Only one task can be attempted per test slot.
 
-	\item \textbf{Participation is default:} Teams have to indicate to the \iaterm{Organizing Committee}{OC} when they are \emph{skipping} a test slot. Without such indication, they may receive a penalty when not attending (see~\refsec{rule:not_attending}).
+	\item \textbf{Participation is default:} Teams have to indicate to the \iaterm{Organizing Committee}{OC} when they are \emph{skipping} a testing slot. Without such indication, they may receive a penalty when not attending (see~\refsec{rule:not_attending}).
```
Comment (MatthijsBurgh) [inline general_rules/Organization.tex]: Maybe a bit out of scope. But why 'may receive a penalty'? Just be strict.

Comment (kyordhel) [reply general_rules/Organization.tex]: Traditionally we apply a penalty for not attending. It is unacceptable not be there. You have to notify OC one hour in advance.

**Comment by kyordhel** [comment]
@benatuts What you see in the rulebook is an example. The OC can modify those slots and the "freedom" to pick up an scenario will depend on logistic constraints as well as how teams are shuffled. It may be the case that the hybrid blocks takes place on the first day starting with Party Host and then Housekeeper.

**Pull request merged**